gotty.go

```diff
@@ -21,33 +22,29 @@ import (
 // If something went wrong reading the terminfo database file, an error is
 // returned.
 func OpenTermInfo(termName string) (*TermInfo, error) {
-	var term *TermInfo
-	var err error
 	// Find the environment variables
-	termloc := os.Getenv("TERMINFO")
-	if len(termloc) == 0 {
+	if termloc := os.Getenv("TERMINFO"); len(termloc) > 0 {
+		path := filepath.Join(termloc, string(termName[0]), termName)
```

Comment:
My worry is that when running in, e.g. `git bash`, I _think_ it would pick back-slashes as a separator, but the environment expects forward slashes. It's a change in behaviour at least (existing code hard-codes `/`, and everything else assumes forward slashes)

@jhowardmsft @rneugeba to the rescue on what is used in that situation :blush: